doc/source/formats/reference/in.rst

```diff
@@ -0,0 +1,12 @@
+.. -*- coding: utf-8 -*-
+.. _IN-format:
+
+=======================================
+IN, FHIAIMS (FHI-aims input files)
+=======================================
+
+.. include:: classes/IN.txt
+
+`FHI-aims`_ input files are similar to :ref:`xyz <_XYZ-format>` in format. The specification is publically available at `online <https://doi.org/10.6084/m9.figshare.12413477.v1>`_. 
+
+.._`FHI-aims`: https://aimsclub.fhi-berlin.mpg.de/
```

Comment:
This does not render properly, maybe it should be  
```
`FHI-aims`_ 
```
rather than
```
_`FHI-aims`
```
but I'm not quite sure about sphinx's syntax.

Reply:
Thanks for pointing that out! Turns out I was missing a space :-)